test/spec-testing.js

```diff
@@ -0,0 +1,69 @@
+// Copyright IBM Corp. 2017. All Rights Reserved.
+// Node module: @loopback/loopback-next-hello-world
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+'use strict';
+
+const Dredd = require('dredd');
+const expect = require('@loopback/testlab').expect;
+const HelloWorldApp = require('../lib/application').HelloWorldApp;
+const http = require('http');
+
+const port = 3030; // use this port for Dredd testing only
+const localhost = 'http://localhost:' + port;
+const dreddConfig = {
+  server: localhost + '/', // base path to the end points
+  options: {
+    level: 'verbose',
+    silent: true, // false for helpful debugging info
+    path: [localhost + '/openapi.json'], // to download apiSpec from the service
```

Comment:
Dredd takes either filepath or  URL to apiSpec.  Chose the latter because did not want to create a temporary file and delete it.  One extra HTTP call is acceptable and the code is cleaner.
@bajtos Do you see any clear win for createing&deleting temporary file  over one HTTP call?